contracts/deployables/strategies/LinearERC20VotingV1.sol

```diff
@@ -116,6 +120,7 @@ contract LinearERC20VotingV1 is
         if (address(_governanceToken) == address(0))
             revert InvalidTokenAddress();
         governanceToken = IVotes(_governanceToken);
+        _clockMode = ClockModeLib.getClockMode(_governanceToken);
```

Comment:
This maybe shouldn't be dynamic, and we should be explicit with an input variable, because the client calling this function will need to know what clock mode governance token is in anyway, in order to give a reasonable value for votingPeriod

Reply:
nope, we're just going to use timestamps for voting periods across the board